app/index.js

```diff
@@ -45,24 +45,24 @@ function buildJSON(investTag) {
     "nid":"14056",
     "links": [
       {
-        "title": `Download InVEST ${latestTag} (Windows) - new Workbench!`,
+        "title": `Download InVEST ${latestTag} (Windows) - Workbench`,
         "url": `${baseUrl}/invest/${latestTag}/workbench/invest_${latestTag}_workbench_win32_x64.exe`
       },
       {
-        "title": `Download InVEST ${latestTag} (Mac) - new Workbench!`,
+        "title": `Download InVEST ${latestTag} (Mac) - Workbench`,
         "url": `${baseUrl}/invest/${latestTag}/workbench/invest_${latestTag}_workbench_darwin_x64.dmg`
       },
       {
-        "title": `Download InVEST ${latestTag} (Windows) - classic application`,
-        "url": `${baseUrl}/invest/${latestTag}/InVEST_${latestTag}_x64_Setup.exe`
+        "title": "InVEST User's Guide (English)",
+        "url": `${baseUrl}/invest-userguide/latest/en/index.html`
       },
       {
-        "title": `Download InVEST ${latestTag} (Mac) - classic application`,
-        "url": `${baseUrl}/invest/${latestTag}/InVEST_${latestTag}.${macExt}`
+        "title": "Guía de uso de InVEST (Español)",
+        "url": `${baseUrl}/invest-userguide/latest/es/index.html`
       },
       {
-        "title": "InVEST User's Guide (online)",
-        "url": `${baseUrl}/invest-userguide/latest/index.html`
+        "title": "InVEST用户指南 (中国人)",
```

Comment:
```suggestion
        "title": "InVEST用户指南 (中文)",
```
Suggesting the translation of "Chinese" that we're getting from pybabel and use in the workbench language dropdown